pom.xml

```diff
@@ -269,6 +277,13 @@
           <groupId>org.sonatype.sisu</groupId>
           <artifactId>sisu-guava</artifactId>
         </exclusion>
+        <exclusion>
+          <!-- promoted-builds requests 1.9.2, core requests 1.8.4 -->
+          <!-- use core version, rather than promoted builds version -->
+          <!-- trust this optional reference will be harmless -->
```

Comment:
Maybe you are just using an old version? Current version picks up 1.8.4 from core.